code/modules/food&drinks/drinks/drinks.dm

```diff
@@ -110,7 +110,7 @@
 	spillable = 1
 
 /obj/item/weapon/reagent_containers/food/drinks/golden_cup/tournament_26_06_2011
-	desc = "A golden cup. It will be presented to a winner of tournament 26 june and name of the winner will be graved on it."
+	desc = "A golden cup. It will be presented to a winner of tournament 26 june and name of the winner will be graved on it. For some reason, it's still not engraved."
 
```

Comment:
No I mean change "graved" to "engraved"

Or maybe just let it have a generic description so admins can hand it out whenever.